app/models/manageiq/providers/amazon/network_manager/refresh_parser.rb

```diff
@@ -350,14 +355,16 @@ def parse_load_balancer_health_check_member(member)
   end
 
   def parse_floating_ip(ip)
-    address = uid = ip.public_ip
+    cloud_network_only = ip.domain["vpc"] ? true : false
```

Comment:
This looks a bit strange. If this is a vpcAddress, then `ip.domain` should return a string and not hash

http://docs.aws.amazon.com/sdkforruby/api/Aws/EC2/VpcAddress.html